next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
```

Comment:
Not exactly sure what changed to make this update happen, but Next made it. My guess is when updating the frontend to no longer use `GetServerSideProps` this file was regenerated as we have no exported types from next